UI/src/view/point_asset/servicePointForm.js

```diff
@@ -106,7 +108,8 @@
       });
     };
 
-    this.renderValueElement = function(asset) {
+    this.renderValueElement = function(asset, collection, authorizationPolicy) {
```

Comment:
Unused definition warning on IntelliJ.

Reply:
The unused function warning is appearing because there is no direct call for this method in the current asset. However this will be call by it's parent in PointAssetForm